### ISSUE TITLE
openapi-generator creates invalid python code when removing underscore from names

### ISSUE THREAD

**Issue by cunningr** [OpenAPITools/openapi-generator#1227]
Description

There seem to many other examples of this for other languages but for python client generation the option of modelPropertyNaming=original has no effect.

We have a vendor published schema that uses item names such as '_id'. To make things worse, within the same object, they use both 'id' and '_id' which after client code generation in python results in;

Traceback (most recent call last):
  File "./example_get_devices.py", line 5, in <module>
    import dnac_api_client
  File "/Users/cunningr/git-projects/dnac-api-client/dnac_api_client/__init__.py", line 19, in <module>
    from dnac_api_client.api.authentication_api import AuthenticationApi
  File "/Users/cunningr/git-projects/dnac-api-client/dnac_api_client/api/__init__.py", line 6, in <module>
    from dnac_api_client.api.authentication_api import AuthenticationApi
  File "/Users/cunningr/git-projects/dnac-api-client/dnac_api_client/api/authentication_api.py", line 21, in <module>
    from dnac_api_client.api_client import ApiClient
  File "/Users/cunningr/git-projects/dnac-api-client/dnac_api_client/api_client.py", line 27, in <module>
    import dnac_api_client.models
  File "/Users/cunningr/git-projects/dnac-api-client/dnac_api_client/models/__init__.py", line 113, in <module>
    from dnac_api_client.models.get_pn_p_global_settings_response import GetPnPGlobalSettingsResponse
  File "/Users/cunningr/git-projects/dnac-api-client/dnac_api_client/models/get_pn_p_global_settings_response.py", line 62
    def __init__(self, sava_mapping_list=None, task_time_outs=None, tenant_id=None, aaa_credentials=None, default_profile=None, accept_eula=None, id=None, id=None, version=None):  # noqa: E501
    ^
SyntaxError: duplicate argument 'id' in function definition
Swagger-codegen version

swagger-codegen version
2.3.1
Swagger declaration file content or url

  GetPnPGlobalSettingsResponse:
    type: object
    properties:
      savaMappingList:
        type: array
        items:
          type: object
          properties:
            syncStatus:
              type: string
            syncStartTime:
              type: number
            syncResult:
              type: object
              properties:
                syncList:
                  type: array
                  items:
                    type: object
                    properties:
                      syncType:
                        type: string
                      deviceSnList:
                        type: array
                        items:
                          type: string
                syncMsg:
                  type: string
            lastSync:
              type: number
            tenantId:
              type: string
            profile:
              type: object
              properties:
                port:
                  type: number
                addressIpV4:
                  type: string
                addressFqdn:
                  type: string
                profileId:
                  type: string
                proxy:
                  type: boolean
                makeDefault:
                  type: boolean
                cert:
                  type: string
                name:
                  type: string
            token:
              type: string
            expiry:
              type: number
            ccoUser:
              type: string
            smartAccountId:
              type: string
            virtualAccountId:
              type: string
            autoSyncPeriod:
              type: number
            syncResultStr:
              type: string
      taskTimeOuts:
        type: object
        properties:
          imageDownloadTimeOut:
            type: number
          configTimeOut:
            type: number
          generalTimeOut:
            type: number
      tenantId:
        type: string
      aaaCredentials:
        type: object
        properties:
          password:
            type: string
          username:
            type: string
      defaultProfile:
        type: object
        properties:
          fqdnAddresses:
            type: array
            items:
              type: string
          proxy:
            type: boolean
          cert:
            type: string
          ipAddresses:
            type: array
            items:
              type: string
          port:
            type: number
      acceptEula:
        type: boolean
      id:
        type: string
      _id:
        type: string
      version:
        type: number

openapi-generator seems to remove the '_', which can make the fields non-unique.

The complete test schema (which does have other non fatal errors) can be found here;

```https://pubhub.devnetcloud.com/media/testdna-center-api-125/docs/swagger_dnacp_125_devnet.json```

Command used to generate;

java -jar openapi-generator-cli.jar generate -i ./dnac1.2.5_swagger.yml -g python -o ~/dnac-api-client -D packageName=dnac_api_client --skip-validate-spec

**Comment by lowdewijk** [comment]
I have the exact same problem using version openapi-generator-cli `4.0.0`. 

I did some digging and I can see why this is happening:

In `PythonClientCodegen.java` the overriden `toVarName` method calls `sanitizeName` which is defined on `DefaultCodegen.java`. This method basically strips everything from the variable name that is does not conform with the `\W` regex, in other words everything that is not `[a-zA-Z0-9]` will be dropped from the variable name. This is done without regard for name collisions. 

Thus, any two model properties that are equal after you strip all non-alphanumeric characters will cause this problem. I have tested this with a number of properties and my hypothesis seems to hold, e.g. `te$st` and `test` will collide.
